apps/meteor/ee/app/livechat-enterprise/server/api/units.ts

```diff
@@ -5,7 +5,7 @@ import { findAllDepartmentsAvailable, findAllDepartmentsByUnit } from '../lib/De
 
 API.v1.addRoute(
 	'livechat/units/:unitId/monitors',
-	{ authRequired: true },
+	{ authRequired: true, permissionsRequired: ['manage-livechat-units'] },
```

Comment:
Since we have a permission check here, so can we remove it from the below method to avoid re-checking?

Reply:
Also, Incorrect permission: this should be `manage-livechat-monitors`